.changeset/new-pets-repair.md

```diff
@@ -0,0 +1,6 @@
+---
+"@apollo/utils.apollokeyv": major
+"@apollo/utils.keyvaluecache": major
+---
+
+Introduce KV cache interface and Keyv adapter
```

Comment:
Maybe be more explicit like "New package which extracts KeyValueCache from Apollo Server and adds a Keyv adapter"?